src/main/java/ru/clevertec/userservice/user/model/request/CreateUserRequest.java

```diff
@@ -15,5 +12,6 @@ public class CreateUserRequest {
     private String lastName;
     private String email;
     private String password;
+    private boolean isActivated;
     private Role role;
```

Comment:
replace with roleId and separate requests for different roles

Reply:
fixed